docker/ray-deps/Dockerfile

```diff
@@ -9,6 +9,7 @@ ENV LANG=C.UTF-8
 COPY $WHEEL_PATH .
 RUN $HOME/anaconda3/bin/pip --no-cache-dir install $(basename $WHEEL_PATH)[all] \
     $(if [ "$AUTOSCALER" = "autoscaler" ]; then echo \
+        "six==1.13.0" \
```

Comment:
Out of curiosity, what is this for?

Reply:
Hi @ijrsvt, `google-api-core` requires `six >= 1.13.0`.